config/redirects.yml

```diff
@@ -11,11 +11,11 @@ redirects:
   "/guidance": "/train-to-be-a-teacher"
 
   # Top-level pages
-  "/teachers-salaries-and-benefits": "/salaries-and-benefits"
+  "/teachers-salaries-and-benefits": "/is-teaching-right-for-me/salaries-and-benefits"
```

Comment:
while we're changing the URL, does it make sense to tweak the /salaries-and-benefits part too, so it's more in line with the H1/page title?